packages/commands/src/edit-track.mjs

```diff
@@ -32,7 +34,9 @@ program.command("edit-track")
         license: license || track.license,
         is_stream_gated: streamConditions ? true : track.is_stream_gated,
         stream_conditions: streamConditions ? JSON.parse(streamConditions) : track.stream_conditions,
-        preview_start_seconds: previewStartSeconds ? parseInt(previewStartSeconds) : track.preview_start_seconds
+        preview_start_seconds: previewStartSeconds ? parseInt(previewStartSeconds) : track.preview_start_seconds,
+        is_unlisted: visibility ? visibility === 'hidden' : track.is_unlisted,
```

Comment:
why the undefined check in `is_private` for playlist and `remix_of` below but not for this line?